XRTK-Core/Packages/com.xrtk.core/Extensions/TransformExtensions.cs

```diff
@@ -338,8 +338,9 @@ public static bool TryGetDepth(Transform target, Transform parent, ref int depth
         /// <param name="transform"></param>
         /// <param name="direction"></param>
         /// <param name="bounds"></param>
+        /// <param name="cachedColliders"></param>
         /// <returns></returns>
-        public static Vector3 GetPointOnBoundsEdge(this Transform transform, Vector3 direction, Bounds bounds = default)
+        public static Vector3 GetPointOnBoundsEdge(this Transform transform, Vector3 direction, Bounds bounds = default, Collider[] cachedColliders = null)
```

Comment:
All input parameters documentation is missing.

Reply:
likely just needs to be copy pasted from above